includes/Admin/Orders.php

```diff
@@ -88,9 +88,11 @@ public function enqueue_assets() {
 		], \WC_Facebookcommerce::VERSION );
 
 		wp_localize_script( 'wc-facebook-commerce-orders', 'wc_facebook_commerce_orders', [
-			'order_id'          => $order->get_id(),
-			'is_commerce_order' => Commerce\Orders::is_commerce_order( $order ),
-			'shipment_tracking' => $order->get_meta( '_wc_shipment_tracking_items', true ),
+			'order_id'                  => $order->get_id(),
+			'order_status'              => $order->get_status(),
+			'is_commerce_order'         => Commerce\Orders::is_commerce_order( $order ),
+			'shipment_tracking'         => $order->get_meta( '_wc_shipment_tracking_items', true ),
+			'allowed_commerce_statuses' => [ 'wc-pending', 'wc-processing', 'wc-completed', 'wc-refunded', 'wc-cancelled' ],
```

Comment:
Also curious as to why you added this param, instead of having this list on the JS.

Reply:
To pipe that into WP Enqueue localization workflow as well as be able to easily change it down the road by `apply_filters` and/or loading the list based on some condition in the backend

Reply:
Ok, fair enough.

Reply:
For this param.